src/app/pages/ArticlePage/experimentTopStories/helpers.tsx

```diff
@@ -23,7 +23,7 @@ type VariantNames = 'Quarter' | 'Half' | 'ThreeQuarters';
 type Positions = 'articleBody' | 'secondaryColumn';
 type TrackingEventType = 'view' | 'click';
 
-const ARTICLE_LENGTH_THRESHOLD = 10;
+const ARTICLE_LENGTH_THRESHOLD = 14;
```

Comment:
Found some issues with some articles between 10 and 14 blocks long during testing, so we're just upping this limit a bit as the easiest/quickest fix. Shouldn't impact our experiment/any other code.